### PR TITLE
1. fix scalars.data is undefined when trying to get image data from s…

### PR DESCRIPTION
**Pull request by BeautyBlack**
…calar object
  -- object [scalars] provides get/set member functions to get/set its internal parameter,  not export its internal parameters directly ;

2. fix mistake setting ImageDataToCornerstoneImage rows&columns values
  -- According Dicom Standar, ROWS means image height, COLUMNS means image width
  -- Dimensions in vtkImageData encode order is [row, column, z-depth]; so as spacing value in vtkImageData.

### CODE REVIEW

**Comment by jourdain** [comment]
Please rewrite commit message to be `fix(ImageDataToCornerstoneImage): swap rows & columns`

**Comment by BeautyBlack** [comment]
Commit message has been rewritten,. It's that ok and wait for review ? @jourdain

**Comment by jourdain** [comment]
I would like to have either @floryst or @thewtex to comment.

File: Sources/Filters/Cornerstone/ImageDataToCornerstoneImage/index.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ function vtkImageDataToCornerstoneImage(publicAPI, model) {
     // FIXME probably need to expand to RGBA
     let pixelData = null;
     if (dims[2] === 1) {
-      pixelData = scalars.data;
+      pixelData = (scalars.data === null || scalars.data === undefined) ? rawData : scalars.data;
```
Comment (floryst) [inline Sources/Filters/Cornerstone/ImageDataToCornerstoneImage/index.js]: This can be simplified to `!scalars.data ? rawData : scalars.data`.

**Comment by floryst** [comment]
Looks good to me overall. The automated checks are red from formatting, so making my suggested change should fix the issue. A handy trick is to run `npm run reformat` before pushing so your code changes will be properly formatted.

**Comment by BeautyBlack** [comment]
I have format codes by pretiier, by still get the failing build reports. I have no idea what to do next now.  Build success on windowsNodes but fail on other nodes,  that confuse me a bit.  I code on windows platform , is it matters?   @floryst

File: Sources/Filters/Cornerstone/ImageDataToCornerstoneImage/index.js [hunk 2]
```diff
@@ -37,11 +37,11 @@ function vtkImageDataToCornerstoneImage(publicAPI, model) {
     // FIXME probably need to expand to RGBA
     let pixelData = null;
     if (dims[2] === 1) {
-      pixelData = scalars.data;
+      pixelData = !scalars.data ? rawData : scalars.data;
     } else {
       const offset =
         model.sliceIndex * dims[0] * dims[1] * rawData.BYTES_PER_ELEMENT;
-      pixelData = new macro.TYPED_ARRAYS[(scalars.getDataType())](
+      pixelData = new macro.TYPED_ARRAYS[scalars.getDataType()](
```
Comment (floryst) [inline Sources/Filters/Cornerstone/ImageDataToCornerstoneImage/index.js]: Try running `npm run reformat` again. The style linter is complaining about this line, and is asking for you to surround the property access like so: `[(scalars.getDataType())]`.

**Comment by BeautyBlack** [comment]
Thank you very much. It pass all checks. @floryst

**Pull request merged**